### PR TITLE
FORMS-16943: added dateToDaysSinceEpoch method

### PR DESCRIPTION
**Pull request by arunattri26**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.frontend/package-lock.json [hunk 1]
```diff
@@ -70,9 +70,9 @@
       }
     },
     "node_modules/@aemforms/af-custom-functions": {
-      "version": "1.0.10",
-      "resolved": "https://registry.npmjs.org/@aemforms/af-custom-functions/-/af-custom-functions-1.0.10.tgz",
-      "integrity": "sha512-n3w9tHkJOI5ISVYAK2cCi5k/oTu3rGgByDmMIgOH1+Ry4mL9nM3cxBTKEkPF8Y8JiKF1aUHIKM+MeP6u5PiiUA=="
+      "version": "1.0.12",
+      "resolved": "https://artifactory.corp.adobe.com/artifactory/api/npm/npm-aem-release/@aemforms/af-custom-functions/-/af-custom-functions-1.0.12.tgz",
```
Comment (rismehta) [inline ui.frontend/package-lock.json:74]: This URL should points to npmjs

Comment (arunattri26) [reply ui.frontend/package-lock.json]: Done.

**Review by rismehta — CHANGES_REQUESTED** [review]
Can you also add a test case for this use-case in this PR via visual rule editor ?

**Comment by arunattri26** [comment]
> Can you also add a test case for this use-case in this PR via visual rule editor ?

I will add it after it is merged. Since rule-editor would expect it to be present else the rule will fail. So we can't release rule-editor without this function being present in core-components.

**Comment by rismehta** [comment]
> > Can you also add a test case for this use-case in this PR via visual rule editor ?
> 
> I will add it after it is merged. Since rule-editor would expect it to be present else the rule will fail. So we can't release rule-editor without this function being present in core-components.

The rule editor shouldn't rely on the core component version, as it may also fail with older versions. Please include test cases in this PR and allow them to fail until the new version of the rule editor SPA is released.

**Comment by arunattri26** [comment]
> > > Can you also add a test case for this use-case in this PR via visual rule editor ?
> > 
> > 
> > I will add it after it is merged. Since rule-editor would expect it to be present else the rule will fail. So we can't release rule-editor without this function being present in core-components.
> 
> The rule editor shouldn't rely on the core component version, as it may also fail with older versions. Please include test cases in this PR and allow them to fail until the new version of the rule editor SPA is released.

Done. I have added the tests for latest-addon and only my authoring test is failing.

**Comment by rismehta** [comment]
@arunattri26 One of the test case is failing, please check and fix it

**Comment by arunattri26** [comment]
> @arunattri26 One of the test case is failing, please check and fix it

@rismehta , I checked it is failing due to old quickstart we are using in tests which does not have the latest exp-editor. After updating the QS with latest devrelease/sites-rotary branch it will get resolved.

**Pull request merged**